packaging/macos/Portfile

```diff
@@ -63,3 +63,4 @@ notes-append "Run @PROJECT_NAME@ by executing 'sunshine <path to user config>',
 notes-append "The config file will be created if it doesn't exist."
 notes-append "It is recommended to set a location for the apps file in the config."
 notes-append "See our documentation at 'https://docs.lizardbyte.dev/projects/sunshine/en/v@PROJECT_VERSION@/' for further info."
+notes-append "For gamepad support, install VirtualHID from 'https://github.com/kotleni/VirtualHID-macOS/releases/latest'
```

Comment:
please, move up one line (just above the documentation link) and leave a blank new line at the end of the file